package.json

```diff
@@ -8,12 +8,16 @@
     "node": ">=10.0.0"
   },
   "scripts": {
-    "build": "scripts/build.sh"
+    "build": "scripts/build.sh",
+    "test": "jest"
   },
   "devDependencies": {
     "@types/aws-lambda": "^8.10.20",
+    "@types/jest": "^26.0.20",
     "@types/node": "12.12.7",
-    "typescript": "^3.3.3333"
+    "jest": "^26.6.3",
+    "ts-jest": "^26.5.3",
+    "typescript": "^3.9.9"
```

Comment:
Any reason to not move to v4? We can do this in a following PR too.

Reply:
This wasn't actually a deliberate change - I guess it must have been bumped automatically when I was installing new dependencies 🤔 

> We can do this in a following PR too.

Yeah, a major version bump probably feels worthy of its own PR, so I will make that change separately?

Reply:
https://github.com/guardian/cloudwatch-logs-management/pull/45